CONTRIBUTING_EN.md

```diff
@@ -10,73 +10,97 @@ Thanks for taking the time to contribute! 🙇‍♀️🙇‍♂️ Every littl
 How to make your first contribution:
 
 - [1. Create a GitHub Account](#1-create-a-github-account)
-- [2. Find an Issue to Work With](#2-find-an-issue-to-work-with)
+- [2. Find an Issue to Work On](#2-find-an-issue-to-work-on)
 - [3. Install Git](#3-install-git)
 - [4. Fork the Project](#4-fork-the-project)
 - [5. Clone your Fork](#5-clone-your-fork)
 - [6. Create a New Branch](#6-create-a-new-branch)
-- [7. Run brutils locally](#7-run-brutils-locally)
-- [8. Make your changes](#8-make-your-changes)
-- [9. Test your changes](#9-test-your-changes)
-- [10. Commit and push your changes](#10-commit-and-push-your-changes)
-- [11. Add changelog entries](#11-add-changelog-entries)
-- [12. Create a GitHub PR](#12-create-a-github-pr)
-- [13. Update your branch if needed.](#13-update-your-branch-if-needed)
+- [7. Run brutils Locally](#7-run-brutils-locally)
+- [8. Make your Changes](#8-make-your-changes)
+- [9. Test your Changes](#9-test-your-changes)
+- [10. Update READMEs](#10-update-readmes)
+- [11. Commit and push your Changes](#10-commit-and-push-your-changes)
+- [12. Add changelog Entries](#11-add-changelog-entries)
+- [13. Create a GitHub PR](#12-create-a-github-pr)
+- [14. Update your branch if Needed.](#13-update-your-branch-if-needed)
 
 ### 1. Create a GitHub Account
 
-Make sure you have a [GitHub account][github-join] and you are logged in with it.
+Ensure you have a [GitHub account][github-join] and you are logged in.
+
+If you don't have an account, 
+follow the steps on [how to create a personal GitHub account][github-essentials-create-account].
 
 ### 2. Find an Issue to Work With
```

Comment:
Na linha 13 (referência), o texto foi alterado para "- [2. Find an Issue to Work On](#2-find-an-issue-to-work-on)" 
```suggestion

### 2. Find an Issue to Work On
```